app/models/doubly_linked_list.rb

```diff
@@ -29,12 +31,16 @@ def each(&block)
     head&.each(&block)
   end
 
-  def push(value)
-    if tail.nil?
-      self.head = self.tail = Node.new(value)
-    else
-      self.tail = self.tail.right = Node.new(value, tail)
+  def push(*values)
```

Comment:
I thought about this, but decided that I had not been asked to provide a multi-value #push.  #push (or #<<) is not defined by Enumerable, in Ruby.  It appears to be defined in Array.  But, there, it is defined specifically for one and only one parameter value (I have wished it to take a splat-list, many times in the past).

That all said, I can't think of a good reason not to support a splat-list.  It'd be awfully convenient.